test_e2e/expected-structured-output/github/test.html.json

```diff
@@ -1,52 +1,100 @@
 [
   {
+    "type": "Title",
     "element_id": "218722ac66e142a570ab2053b430c6c4",
+    "text": "Downloadify Example",
     "metadata": {
-      "filetype": "text/html",
       "languages": [
         "eng"
-      ]
-    },
-    "text": "Downloadify Example",
-    "type": "Title"
+      ],
+      "filetype": "text/html",
+      "data_source": {
+        "url": "https://api.github.com/repos/dcneiner/Downloadify/git/blobs/c63c8fc21d46d44de85a14a3ed4baec0348ce344",
+        "version": "W/\"a75db90ed53f327faf43999f4719f8d200fb19cf1db08fe6ace57e88426ec4f4\"",
+        "record_locator": {
+          "repo_path": "dcneiner/Downloadify",
+          "file_path": "test.html"
+        },
+        "date_created": null,
```

Comment:
As we had discussed. Definitely try to fill these nulls in if that type of information is available. 

Now on the other hand, once you get those to fill in correctly, (we always want as much metadata as possible) then add the line to ignore it in the expected tests.

https://github.com/Unstructured-IO/unstructured-ingest/blob/b3add2abf347b5e21121bf01626f8b4e9e3ec9e1/test_e2e/src/github.sh#L49

Reply:
Perfect, thank you Potter! I've already added it.